pkg/pmem-device-manager/pmd-lvm.go

```diff
@@ -112,6 +112,11 @@ func (lvm *pmemLvm) CreateDevice(name string, size uint64, nsmode string) error
 			if _, err := pmemexec.RunCommand("lvcreate", "-Zn", "-L", strSz, "-n", name, vg.name); err != nil {
 				glog.V(3).Infof("lvcreate failed with error: %v, trying for next free region", err)
 			} else {
+				// Sometimes needed softlinks that are supposed to create by udev are missing,
+				// hence we force create them by using vgscan
```

Comment:
"supposed to be created", missing dot at end of sentence.

Does this `vgscan` now race with udev? Where does this udev run - on the host or in the container?

I'm asking because we cannot know whether the host system has a running udev. I'm wondering whether we have a (so far) hidden dependency on the host system here.

Reply:
I guess by defaults its using host udev, In my next commit I asked to disable udev syncing in container, so there should not be any race conditions with host udev.